### PR TITLE
TW-217: Update new link support page

### PR DESCRIPTION
**Pull request by nqhhdev**
## Ticket
- https://github.com/linagora/twake-workplace-private/issues/217

## Resolved
**Attach screenshots or videos demonstrating the changes**
- Web:

https://github.com/user-attachments/assets/a26beab2-c045-48b0-aa22-795c5917580d

- IOS:

https://github.com/user-attachments/assets/ca7f9940-4b89-4fce-b412-e9f988277f17

### CODE REVIEW

File: lib/pages/app_grid/app_grid_dashboard_view.dart [hunk 1]
```diff
@@ -19,71 +21,99 @@ class AppGridDashboardView extends StatelessWidget {
...
-                      ),
-                    ),
-                  );
-                },
+              child: MouseRegion(
+                cursor: SystemMouseCursors.click,
+                child: SizedBox(
+                  width: AppGridDashboardViewStyle.sizeSupportButton,
+                  height: AppGridDashboardViewStyle.sizeSupportButton,
+                  child: SvgPicture.asset(
```
Comment (hoangdat) [inline lib/pages/app_grid/app_grid_dashboard_view.dart:41]: Please add tooltip + hover for this icon.

Comment (nqhhdev) [reply lib/pages/app_grid/app_grid_dashboard_view.dart]: Updated 

https://github.com/user-attachments/assets/92df24a7-577e-4a58-9834-3f616ca6f77e

File: lib/pages/app_grid/app_grid_dashboard_view.dart [hunk 2]
```diff
@@ -19,71 +22,126 @@ class AppGridDashboardView extends StatelessWidget {
...
+                        message: L10n.of(context)!.help,
+                        showDuration: const Duration(seconds: 1),
+                        waitDuration: const Duration(seconds: 1),
+                        child: Container(
+                          decoration: BoxDecoration(
+                            color: color,
+                            borderRadius: AppGridDashboardViewStyle
+                                .appGridIconBorderRadius,
+                          ),
+                          padding: AppGridDashboardViewStyle.appGridIconPadding,
```
Comment (hoangdat) [inline lib/pages/app_grid/app_grid_dashboard_view.dart:62]: it makes our icon so small.

image

Comment (nqhhdev) [reply lib/pages/app_grid/app_grid_dashboard_view.dart:62]: IMO: My fault when exporting the icon included background color was `White`, I updated it. pls verify it again.

Comment (nqhhdev) [reply lib/pages/app_grid/app_grid_dashboard_view.dart:62]: https://github.com/user-attachments/assets/cb6d7516-0da8-4d3f-b6cc-9e23ad1517fe

**Review by hoangdat — APPROVED** [review]

**Pull request merged**